data_model/src/query.rs

```diff
@@ -252,16 +256,18 @@ pub mod role {
         PartialOrd,
         Ord,
     )]
+    #[display(fmt = "Find all roles")]
```

Comment:
Consider making this the Upper-camel-case name of struct.

Reply:
I like how it is now. Because `UpperCamelCase` won't look good with other displays like:

```rust
#[display(fmt = "Find `{}` role", id)]
```

Reply:
How about 
```rust
#[display(fmt = "FindRoleById {{ id: {} }}" id)]
```

In fairness this looks more like `Debug`, so another alternative would be 
```rust 
#[display(fmt = "FindRoleById: `{}`", id)]
```

Reply:
Stil unclear what to do with i.e. `FindAccountKeyValueByIdAndKey`, which has 2 fields.
Yes, the first version looks more like `Debug` and I wanted to make an error message more human-redable

Reply:
We can revisit this later.